src/frontend/services/dataServices.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { dataConstants } from "../constants";
+
+const getVideos = async (dispatch) => {
+	try {
+		const response = await axios.get("/api/videos");
+		dispatch({
```

Comment:
suggestion: u can extract all the URL in a separate file

Reply:
will do that